src/prefect/task_worker.py

```diff
@@ -339,6 +382,19 @@ def yell(message: str):
     """
     task_worker = TaskWorker(*tasks, limit=limit)
 
+    loop = asyncio.get_event_loop()
+
+    server = uvicorn.Server(
+        uvicorn.Config(
+            app=create_status_server(task_worker),
+            host="127.0.0.1",
+            port=4422,
```

Comment:
Should we grab a random port to avoid failures when starting multiple task workers on the same instance?